make/ocm.mk

```
@@ -75,15 +75,15 @@ ocm/cluster/upgrade:
 ocm/help:
 	@${OCM_SH} -h
 
-.PHONY: ocm/provision_sts_cluster
-ocm/sts/cluster/create:
-	@${STS_SH} provision_sts_cluster
+.PHONY: ocm/rosa/cluster/create
+ocm/rosa/cluster/create:
+	@${ROSA_SH} provision_sts_cluster
 
-.PHONY: ocm/delete_sts_cluster
-ocm/sts/cluster/delete:
-	@${STS_SH} delete_sts_cluster
+.PHONY: ocm/rosa/cluster/delete
+ocm/rosa/cluster/delete:
+	@${ROSA_SH} delete_sts_cluster
 
 .PHONY: ocm/rhoam-prerequisites
```

```suggestion
.PHONY: ocm/sts/rhoam-prerequisites
```

Minor, but lets fix this also while we're updating the makefile anyway 👍